nvme.c

```diff
@@ -6674,6 +6679,177 @@ static int show_hostnqn_cmd(int argc, char **argv, struct command *command, stru
 	return 0;
 }
 
+#ifdef CONFIG_OPENSSL_1
+static int gen_dhchap_key_ossl(char *hostnqn, unsigned int hmac,
```

Comment:
Only one comment: shouldn't this function go into libnvme?

Reply:
Done.